app/assets/stylesheets/table-category-column.scss

```diff
@@ -5,7 +5,3 @@
     visibility: visible;
   }
 }
```

Comment:
could we consolidate all table styles within `app/assets/stylesheets/tables.scss`